Update sql-database-export.md

GoEddie opened the pull request:
if you check for status then sleep you don't give it a chance to complete so end up with an extra sleep :)

azurecla commented:
Hi __@mwinkle__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (mwinkle). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

GoEddie committed:
Update sql-database-export.md

if you check for status then sleep you don't give it a chance to complete so end up with an extra sleep :)
